Cyra/lab2/part2/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.32)
+project(hello_lib)
+
+add_library(hello_lib STATIC src/hello.cpp)
+
+
+
+target_include_directories(hello_lib PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
+    $<INSTALL_INTERFACE:src>
+)
```

Comment:
instead of `src` this include directory that is being pointed to should be something like your include, or wherever your header files are located. by convention this should be `include`